2021-04-27T17:29:20.1834180Z ##[group]Run mypy tinuous
2021-04-27T17:29:20.1834681Z [36;1mmypy tinuous[0m
2021-04-27T17:29:20.1872698Z shell: /usr/bin/bash -e {0}
2021-04-27T17:29:20.1873058Z env:
2021-04-27T17:29:20.1873534Z   pythonLocation: /opt/hostedtoolcache/Python/3.8.9/x64
2021-04-27T17:29:20.1874164Z   LD_LIBRARY_PATH: /opt/hostedtoolcache/Python/3.8.9/x64/lib
2021-04-27T17:29:20.1874642Z ##[endgroup]
2021-04-27T17:29:24.3688528Z Success: no issues found in 1 source file
